apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1232,6 +1521,15 @@ static void tc_fs_vfs_rename(void)
 	ret = rename(old_file, NULL);
 	TC_ASSERT_EQ("rename", ret, ERROR);
 
+
+	/*Condition where rename is not possible*/
+	ret = rename("/dev/null", "/dev/NULL");
```

Comment:
isn't is possible to #define these paths ?

Reply:
yes will add as Macros